pyproject.toml

```
@@ -1,8 +1,23 @@
 [tool.poetry]
 name = "youtrack-sdk"
-version = "0.1.0"
-description = ""
-authors = []
+version = "1.1+SNAPSHOT"
+description = "YouTrack SDK"
+authors = ["moneymeets <admin@moneymeets.com>"]
```

```suggestion
authors = ["moneymeets <service@moneymeets.com>"]
```

My suggestion is to use our Service Desk connected e-mail for the outside world.

---

Done.